_posts/2023-02-13-announcing-duckdb-070.md

```diff
@@ -241,6 +241,8 @@ shape: (1, 1)
 
 **fsspec Filesystem Support.** This release adds support for the [fsspec filesystem API](https://github.com/duckdb/duckdb/pull/5829). [fsspec](https://filesystem-spec.readthedocs.io/en/latest/) allows users to define their own filesystem that they can pass to DuckDB. DuckDB will then use this file system to read and write data to and from. This enables support for storage back-ends that may not be natively supported by DuckDB yet, such as FTP.
 
+Have a look at the [guide](https://duckdb.org/docs/guides/python/filesystems) for more information
```

Comment:
Perhaps it would be nice to add an example to the `fsspec` inline in the blog post, e.g. showing how to connect to GCS or so?